pybamm/models/submodels/interface/open_circuit_potential/plett_ocp.py

```diff
@@ -0,0 +1,157 @@
+            T = variables[f"{Domain} electrode temperature [K]"]
+            h = variables[f"{Domain} electrode {phase_name}hysteresis state"]
+            # For "particle-size distribution" models, take distribution version
+            # of c_s_surf that depends on particle size.
+            domain_options = getattr(self.options, domain)
+            if domain_options["particle size"] == "distribution":
+                sto_surf = variables[
+                    f"{Domain} {phase_name}particle surface stoichiometry distribution"
+                ]
+                # If variable was broadcast, take only the orphan
+                if isinstance(sto_surf, pybamm.Broadcast) and isinstance(
+                    T, pybamm.Broadcast
+                ):
+                    sto_surf = sto_surf.orphans[0]
+                    T = T.orphans[0]
+                T = pybamm.PrimaryBroadcast(T, [f"{domain} particle size"])
+                h = pybamm.PrimaryBroadcast(h, [f"{domain} particle size"])
+            else:
+                sto_surf = variables[
+                    f"{Domain} {phase_name}particle surface stoichiometry"
+                ]
+                # If variable was broadcast, take only the orphan
+                if isinstance(sto_surf, pybamm.Broadcast) and isinstance(
+                    T, pybamm.Broadcast
+                ):
+                    sto_surf = sto_surf.orphans[0]
+                    T = T.orphans[0]
+
+            ocp_surf_eq = self.phase_param.U(sto_surf, T)
+            dUdT = self.phase_param.dUdT(sto_surf)
+
+            # Bulk OCP is from the average SOC and temperature
+            sto_bulk = variables[f"{Domain} electrode {phase_name}stoichiometry"]
+            T_bulk = pybamm.xyz_average(pybamm.size_average(T))
+            ocp_bulk_eq = self.phase_param.U(sto_bulk, T_bulk)
+
+            # # get dQ, change in capacity with respect to stoichiometry
+            # c_max = self.phase_param.c_max
+            # epsilon_s_av = self.phase_param.epsilon_s_av
+            # V_electrode = self.param.A_cc * self.domain_param.L
+            # Li_max = c_max * V_electrode * epsilon_s_av
+            # Q_max = Li_max * self.param.F / 3600
+            # dQ = Q_max
+
+            c_scale = self.phase_param.c_max
+            variables[f"Total lithium in {phase} phase in {domain} electrode [mol]"] = (
+                sto_bulk * c_scale
+            )  # c_s_vol * L * A
+
+            H = self.phase_param.H(sto_surf)
+            variables[f"{Domain} electrode {phase_name}equilibrium OCP [V]"] = (
+                ocp_surf_eq
+            )
+            variables[f"{Domain} electrode {phase_name}bulk equilibrium OCP [V]"] = (
+                ocp_bulk_eq
+            )
+            variables[f"{Domain} electrode {phase_name}OCP hysteresis [V]"] = H
+
+            dU = self.phase_param.U(sto_surf, T_bulk).diff(sto_surf)
+            # dQ = variables[f"{Domain} electrode {phase_name}capacity [A.h]"]
```

Comment:
I tried using the variable here, but it looks like it doesn't exist for individual phases.